spec/lib/portus/ldap_spec.rb

```diff
@@ -304,7 +304,8 @@ def load_configuration_test
       APP_CONFIG["ldap"] = { "enabled" => true, "base" => "" }
       lm = LdapMock.new(username: "", password: "1234")
       lm.authenticate!
-      expect(lm.last_symbol).to be :invalid_username
+      expect(lm.last_symbol).to be "Password is too short (minimum is 8 characters),"\
```

Comment:
This should be changed to `expect(lm.last_symbol).to eq("Password is....")` to make the tests pass